hathor/feature_activation/feature.py

```diff
@@ -16,7 +16,7 @@
 
 
 @unique
-class Feature(Enum):
+class Feature(str, Enum):
```

Comment:
Why this change?

Reply:
Because now we're serializing this to json (as it's stored in the database), so adding this make it work (otherwise it tries to serialize a raw enum and fails). It's equivalent to using `StrEnum` from Python 3.11.

Reply:
Got it. Thanks!